src/App.vue

```diff
@@ -19,13 +19,15 @@
     <div class="flex flex-col mb-auto px-4 py-10">
       <router-view class="text-center" />
     </div>
-    <Message
-      v-for="(message, index) in messages"
-      ref="alert"
-      :key="index"
-      v-bind="message"
-      @dismissed="dismiss(index)"
-    ></Message>
+    <div class="message-container">
```

Comment:
Confines the alerts to the bottom right of the viewport, making them behave like toasts. Increases the z-index to ensure they are easily visible when a modal is present.